### PR TITLE
LPD-32724 Asset Publisher using site's theme instead of default styles

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-32724)

## Changes

Migrate element to a clay component and remove th spritemap to don't use the site's theme

## Test Scenario

1. Create a Page
2. Add an Asset Publisher widget
3. Go to the configuration of the widget and select dynamic or manual collection
4. Click on the new link "Create a collection from this configuration."
5. Create a collection
6. Verify that it has been created.

[Screencast from 2024-08-13 18-27-49.webm](https://github.com/user-attachments/assets/bebcb882-3fc3-463c-a1b0-aea6f71c2c00)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ealonso** [comment]
@BarbaraCabrera you need to add tests

File: modules/test/playwright/tests/asset-publisher-web/configuration.spec.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+		pageEditorPage,
+		'Dynamic'
+	);
+
+	await collectionsPage.goto(site.friendlyUrlPath);
+
+	await expect(page.getByText(collectionName)).toBeVisible();
+});
+
+async function createCollectionFromAssetPublisher(
```
Comment (sandrodw3) [inline modules/test/playwright/tests/asset-publisher-web/configuration.spec.ts:102]: I think the best practice would be to create an `AssetPublisherPage` and move this method there, so in the future anyone can use it if needed, would you mind doing it? 😇

File: modules/test/playwright/tests/asset-publisher-web/configuration.spec.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	await iframe
+		.getByRole('button', {name: 'Create a collection from this'})
+		.click();
+
+	await iframe.getByPlaceholder('Title').fill(collectionName);
+
+	await iframe
+		.getByLabel('Collection Title')
+		.getByRole('button', {name: 'Save'})
+		.click();
```
Comment (sandrodw3) [inline modules/test/playwright/tests/asset-publisher-web/configuration.spec.ts:120]: To make the method more robust, I think we should add here `await waitForSuccessAlert(
		page,
		'Success:The collection was created successfully.'
	);`, so we make sure the action is completed

**Review by sandrodw3 — COMMENTED** [review]
@BarbaraCabrera code looks good and it works perfectly! Just suggesting minor improvements and adjustments 😇

**Comment by BarbaraCabrera** [comment]
I think that I've changed all, let me know if I need to do more improvements and ... thanks a lot! ❤️

**Comment by ealonso** [comment]
ci:forward

**Comment by BarbaraCabrera** [comment]
ci:stop

**Comment by BarbaraCabrera** [comment]
ci:forward

**Comment by BarbaraCabrera** [comment]
Stoping the CI because I need to add [this](https://github.com/liferay/liferay-portal/blob/master/modules/apps/layout/layout-content-page-editor-web/test.properties#L14) and [this](https://github.com/liferay/liferay-portal/blob/master/modules/apps/layout/layout-content-page-editor-web/test.properties#L33)

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:reevaluate:1317401_11271

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**